Cython/Build/__init__.py

```diff
@@ -3,8 +3,17 @@
 import sys
 if sys.version_info < (3, 7):
     from .Distutils import build_ext
-del sys
 
+if sys.version_info > (3, 11):
+    try:
+        import setuptools
+    except ImportError:
+        raise ImportError(
```

Comment:
This handling is repeating all the times and maybe it deserves to be moved to function. But not sure where this function put.

Reply:
It'll probably be removed at some point, and it's simple enough. I'm fine with repeating it.